open-grants/README.md

```diff
@@ -0,0 +1,48 @@
+
+- **Developer tools and libraries**: We are always looking for proposals for the development of developer tools and libraries for both core protocol developers and application developers. Dev tools at all layers of the stack are invaluable! Relevant proposals should be labeled [`category:devtools-libraries`](https://github.com/filecoin-project/devgrants/labels/category%3Adevtools-libraries).
+
+- **Technical design**: We want ideas that will improve on the Filecoin protocol technical design, for example improvement proposals on the core storage protocol, cryptoeconomics design, or on other planned network upgrades. Relevant proposals should be labeled [`category:technical-design`](https://github.com/filecoin-project/devgrants/labels/category%3Atechnical-design).
+
+- **Documentation**: We understand how crucial documentation is for any open-source project and we welcome proposals to create or improve existing Filecoin project documentation. Relevant proposals should be labeled [`category:docs`](https://github.com/filecoin-project/devgrants/labels/category%3Adocs).
+
+- **Community**: We would like to support community initiatives throughout the global Filecoin ecosystem. We have a standing RFP for hosting community meetups, and are generally excited to hear about all sorts of community initiatives. Relevant proposals should be labeled [`category:community`](https://github.com/filecoin-project/devgrants/labels/category%3Acommunity).
+
+## 📋 How to apply
+                                    
+Here are the steps to submit a proposal for the open grant category!
+1. Submit a PR against the [`open-proposal-template.md` file](https://github.com/filecoin-project/devgrants/tree/master/open-grants/open-proposal-template.md) in this [`open-grants` folder](https://github.com/filecoin-project/devgrants/tree/master/open-grants). Make sure to fill in all the categories of the proposal template. Rename your file `project-title.md`, making sure to replace `project-title` with the name of your project.
+                                                                                                                        
+Open Grant proposals are reviewed monthly. The deadline for the next wave is **August 31, 2021**                                                               
+
+## ⌛ After you apply
+
+After you submit your proposal, you can expect the following to occur:
+
+- We will review your application. During review, we will add comments, questions, change requests, et cetera on your team's submission.
+- After a few round trips of discussion, our team will make a decision on which proposals to fund and which not to. <!--Accepted proposals will be merged into the appropriate directory, i.e. either [`open-grant`](https://github.com/filecoin-project/devgrants/tree/master/open-grant) or [`rfp-proposals`](https://github.com/filecoin-project/devgrants/tree/master/rfp-proposals).-->
```

Comment:
@mosh Not tracking a lot of projects that have merged to the repo, we can add the provision to list on the projects Showcase here as well.